force-app/main/default/lwc/attendanceRow/attendanceRow.js

```diff
@@ -23,27 +26,52 @@ export default class AttendanceRow extends LightningElement {
 
     @api unitOfMeasurement;
     @api presentStatus = PRESENT_STATUS;
-    oldStatus;
+    @api readOnly = false;
     name;
+    _isEdited;
+    recordId;
 
     @api
     get record() {
         return this.localRecord;
     }
     set record(value) {
+        // force lightning data service to refresh its cache so lightning-record-view-form updates its values
```

Comment:
Is this a hack to get around the issue you were working through last night?

Reply:
Also, would the new getRecordNotifyChange work here?

Reply:
ooh maybe i will look into that!